pkgs/top-level/all-packages.nix

```diff
@@ -15475,11 +15477,11 @@ in
   bridge-utils = callPackage ../os-specific/linux/bridge-utils { };
 
   busybox = callPackage ../os-specific/linux/busybox { };
-  busybox-sandbox-shell = callPackage ../os-specific/linux/busybox/sandbox-shell.nix { 
+  busybox-sandbox-shell = callPackage ../os-specific/linux/busybox/sandbox-shell.nix {
```

Comment:
Would you mind splitting the whitespace changes out into a second commit? It is just in case people (in the future) might be confused why a new package touched `busybox`. It might save someone 15min of time wasted due to confusion.